File: samples/client/petstore/go/swagger/InlineResponse200.go
package swagger

import (
)

type InlineResponse200 struct {
    Tags  []Tag  `json:"tags,omitempty"`
    Id  int64  `json:"id,omitempty"`
    Category  Object  `json:"category,omitempty"`
    Status  string  `json:"status,omitempty"`
    Name  string  `json:"name,omitempty"`
    PhotoUrls  []string  `json:"photoUrls,omitempty"`
    
}

File: samples/client/petstore/typescript-angular/API/Client/InlineResponse200.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface InlineResponse200 {

        "tags"?: Array<Tag>;

        "id": number;

        "category"?: any;

        /**
         * pet status in the store
         */
        "status"?: InlineResponse200.StatusEnum;

        "name"?: string;

        "photoUrls"?: Array<string>;
    }

    export namespace InlineResponse200 {

        export enum StatusEnum { 
            available = <any> 'available',
            pending = <any> 'pending',
            sold = <any> 'sold'
        }
    }
}
